core/workspace_svg.js

```diff
@@ -1908,6 +1939,27 @@ Blockly.WorkspaceSvg.prototype.cancelCurrentGesture = function() {
 };
 
 /**
+ * Don't even think about using this function before talking to rachel-fenichel.
+ *
+ * Force a drag to start without clicking and dragging the block itself.  Used
+ * to attach duplicated blocks to the mouse pointer.
+ * @param {!Object} fakeEvent An object with the properties needed to start a
+ *     drag, including clientX and clientY.
+ * @param {!Blockly.BlockSvg} block The block to start dragging.
+ * @package
+ */
+Blockly.WorkspaceSvg.prototype.startDragWithFakeEvent = function(fakeEvent,
+    block) {
+  Blockly.Touch.clearTouchIdentifier();
+  // Also sets the touch identifier
+  if (!Blockly.Touch.checkTouchIdentifier(fakeEvent)) {
+    console.log('something went wrong while setting the touch identifier');
```

Comment:
nit: it looks the non-test code all use `console.warn` instead of `log`, seems like this would deserve a warning. Similar to other error message comment, it might be good to include some hints (if you have any) about why that can fail.